lib/services/platform-service.ts

```diff
@@ -82,7 +82,7 @@ export class PlatformService implements IPlatformService {
 		let packageToInstall = "";
 		let npmOptions: IStringDictionary = {
```

Comment:
can we add `--save-exact` to `npmOptions`, as currently we are adding the framework package with `^`, for example:
```
"tns-android": "^2.5.0"
```
This means that once we have `tns-android` 2.6.0, and I execute `rm -rf node_modules && npm i` I'll get the new framework.